PWGEM/PhotonMeson/Utils/TrackSelection.h

```diff
@@ -0,0 +1,145 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \brief helper functions for pair track selection
+/// \author felix.schlepper@cern.ch
+
+#ifndef PWGEM_PHOTONMESON_UTILS_TRACKSELECTION_H_
+#define PWGEM_PHOTONMESON_UTILS_TRACKSELECTION_H_
+
+namespace o2::pwgem::photonmeson
+{
+
+/**
+ * @brief Track has ITS and TPC
+ *
+ * @tparam TTrack track
+ * @param track track
+ * @return true if has both
+ */
+template <typename TTrack>
+inline bool isITSTPCMatchedTrack(TTrack const& track)
+{
+  return track.hasITS() && track.hasTPC();
+}
+
+/**
+ * @brief Track is TPC-only
+ *
+ * @tparam TTrack track
+ * @param track track
+ * @return true if tracks is TPC-only
+ */
+template <typename TTrack>
+inline bool isTPConlyTrack(TTrack const& track)
+{
+  return !track.hasITS() && track.hasTPC();
```

Comment:
Can you modify return !track.hasITS() && track.hasTPC() && track.hasTRD() && track.hasTOF();